docs/development/minikube.md

```diff
@@ -70,7 +75,7 @@ $ ansible-playbook ansible/instantiate-awx-deployment.yml \
     -e image_version=devel \
     -e image_pull_policy=Always \
     -e service_type=nodeport \
-    -e namespace=default
+    -e namespace=$NAMESPACE
 ```
 
```

Comment:
```suggestion
Check the operator with the following commands:
```
# Check the operator deployment
kubectl get deployments

NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
awx                               1/1     1            1           16h
awx-operator-controller-manager   1/1     1            1           16h

kubectl get pods -n awx                                                                                                                                                                                                             ST 7   consume_control_node_capacity 
NAME                                              READY   STATUS    RESTARTS   AGE
awx-operator-controller-manager-b775bfc7c-fn995   2/2     Running   0          16h
```

If there are errors in the image pull, check that it is using the right tag. You can update the tag that it will pull by editing the deployment.
```